src/ops/init.c

```diff
@@ -86,13 +86,21 @@ static void op_INIT_SCENE_get(const void *NOTUSED(data), scene_state_t *ss,
 static void op_INIT_SCRIPT_get(const void *NOTUSED(data), scene_state_t *ss,
                                exec_state_t *NOTUSED(es), command_state_t *cs) {
     int16_t v = cs_pop(cs) - 1;
-    if (v >= 0 && v < TEMP_SCRIPT) ss_clear_script(ss, (size_t)v);
+    if (v >= 0 && v < TEMP_SCRIPT) {
+        ss_clear_script(ss, (size_t)v);
+        ss->variables.j[v] = 0;
```

Comment:
maybe move this and the next line to `ss_clear_script`?

Reply:
will do